src/java.net.http/share/classes/java/net/http/HttpResponse.java

```diff
@@ -748,6 +749,33 @@ public static <T> BodyHandler<T> buffering(BodyHandler<T> downstreamHandler,
                      .buffering(downstreamHandler.apply(responseInfo),
                                 bufferSize);
          }
+
```

Comment:
The copyright year on this file will need an update.

Reply:
Fixed in 453cd0473b43e3e25cbb8918a575db3b8421af7d.